src/ramda-extended.js

```diff
@@ -39,8 +39,6 @@
   R.rmap = R.curryN(2, function rmap (obj, fns) {
     return R.map(R.applyTo(obj), fns)
   });
-  // @sig [(a -> b)] -> a -> [b]
-  R.juxt = R.flip(R.rmap);
```

Comment:
The new native `R.juxt` method is not a 1:1 replacement for this.

The native implementation: `juxt applies a list of functions to a list of values.` whereas this implementation is `applies a list of functions to a single value`.

Testing some stuff out though, I think it's a forward-compatible change since the new native one is just more flexible.

Our function:
```
> juxt([R.add, R.inc, R.dec])(5, 10, 50)
< [f1(a), 6, 4]
```

Their function:
```
> R.juxt([R.add, R.inc, R.dec])(5, 10, 50)
< [15, 6, 4]
```